etc/compile_check.sh

```diff
@@ -2,40 +2,41 @@
 set -e # exit when any command fails
 set -x # show all commands being run
 
-GC=go
+# Default to Go 1.18 if GO_VERSION is not set.
+#
+# Use the "=" operator (instead of the more common ":-" operator) so that it
+# allows setting GO_VERSION="" to use the Go installation in the PATH, and it
+# sets the GO_VERSION variable if the default is used.
+GC=go${GO_VERSION="1.18"}
```

Comment:
```suggestion
GC=go${GO_VERSION:-"1.18"}
```

Reply:
I'm using `=` because I want these two behaviors:
* Allow setting `GO_VERSION=""`, which prevents setting the default "1.18" and allows using the Go binary in PATH.
* When the default is used, set `GO_VERSION="1.18"` so the following check (`if [ ! -z $GO_VERISON ]`) will be true. Otherwise, `${GC}` will be `go1.18`, but the block that downloads Go 1.18 won't run.

Reply:
Ah okay, I'm just not familiar with the syntax, sounds fine to me.